lib/src/models/continent.dart

```diff
@@ -0,0 +1,26 @@
+part of appwrite.models;
+
+class Continent {
+    late final String name;
```

Comment:
`late` keyword is redundant here
```suggestion
    final String name;
```